### PR TITLE
Allow devices named 'reset' in select-other-device

### PR DESCRIPTION
**Pull request by buoyad**
~Use an invalid device name for the "Reset your account" sentry value.~ Use a different type to represent the reset row.
cc @keybase/y2ksquad

### CODE REVIEW

**Comment by jakob223** [comment]
cc @hyperobject

**Comment by zapu** [comment]
thanks :tada:

**Comment by hyperobject** [comment]
looks great! 🎉

File: shared/provision/select-other-device/index.tsx [hunk 1]
```diff
@@ -11,9 +11,6 @@ type Props = {
   onBack: () => void
 }
 
-// needs to be an invalid device name
-const resetSentry = '!reset'
```
Comment (transparentCF) [inline shared/provision/select-other-device/index.tsx]: Fortunately you've realized your error. The first revision of the fix isn't idiomatic TypeScript, and further usage of this pattern can lead to more reliability issues. Not using the name field at all is the perfect approach. Thanks for the fast response.

**Pull request merged**